onnxruntime/core/graph/function_utils.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "onnx/onnx_pb.h"
+#include "core/graph/graph.h"
+#include "core/graph/indexed_sub_graph.h"
+#include "core/graph/function.h"
+#include "core/graph/schema_registry.h"
+
+namespace onnxruntime {
+namespace function_utils {
+
+std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const Graph& graph,
+                                                       const IndexedSubGraph& nodes_to_fuse);
+
+std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const std::string& function_domain,
+                                                       const std::string& function_name,
+                                                       const std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*>& model_local_functions,
+                                                       const std::unordered_map<std::string, int>& domain_version_map,
```

Comment:
Change to InlinedHashMap

Reply:
I fix the map for function proto because it is introduced in this PR. but i can't change the domain_to_version map, because that one is passing the existing member defined in Model, if i update it, i will need to update the [Model API](https://github.com/microsoft/onnxruntime/blob/91c940b619f1096f730ee1096c5befd4cc3e81d5/onnxruntime/core/graph/model.h#L51) and fix all the places that refer to it. That will make the scope for this pr much bigger. Since the domain_to_version map has been defined in Model for quite a while, i don't think this pr will make things worse.

if it is really a concern, we can have a follow up PR to update the domain_to_version map in Model/Graph.

Reply:
You can still add reserve() calls to it.

Reply:
add the reserve in the caller.